cypress/e2e/pages/topicPage/index.cy.js

```diff
@@ -1,30 +1,71 @@
-import config from '../../../support/config/services';
-import getPaths from '../../../support/helpers/getPaths';
-import serviceHasPageType from '../../../support/helpers/serviceHasPageType';
-
+import runTestsForPage from '#nextjs/cypress/support/helpers/runTestsForPage';
 import crossPlatformTests from './tests';
-import visitPage from '../../../support/helpers/visitPage';
-import { getTopicPagePath } from './helpers';
+import { TOPIC_PAGE } from '../../../../src/app/routes/utils/pageTypes';
```

Comment:
Let's create a variable here for tests, so that we don't need to repeat the same line multiple times:

```
const tests = [crossPlatformTests];
```